doc/config.md

```diff
@@ -1426,6 +1435,30 @@ file from the `pg_shadow` system table.  Alternatively, use
 `auth_query` instead of `auth_file` to avoid having to maintain a
 separate authentication file.
 
+### Notes on managed servers
+If the backend server is configured to use SCRAM password authentication PgBouncer cannot 
+successfully authenticate if it does not know either a) user password in plain text or 
+b) corresponding SCRAM secret.
+
+Some cloud providers (i.e. AWS RDS) prohibit access to PostgreSQL sensitive system tables 
+for fetching passwords. Even for the most privileged user (i.e. member of rds_superuser) the
+`select * from pg_authid`;  returns the `ERROR:  permission denied for table pg_authid.`
+That is a known behaviour 
+([blog](https://aws.amazon.com/blogs/database/best-practices-for-migrating-postgresql-databases-to-amazon-rds-and-amazon-aurora/)).
+
+Therefore, fetching an existing SCRAM secret once it has been stored in a managed server
+is impossible which makes it hard to configure PgBouncer to use the same SCRAM secret. 
+Nevertheless, SCRAM secret can still be configured and used on both sides using the following trick: 
+
+Generate SCRAM secret for arbitrary password locally (using a custom script or local postgres installation 
+running on your desktop). Then take the secret and set it inside PgBouncer's `userlist.txt` 
+as well as inside managed server via alter role command.
+
+```sql
+alter role <role_name> password 'SCRAM-SHA-256$<iterations>:<salt>$<storedkey>:<serverkey>';
+```
```

Comment:
I think the easiest and most secure way to do this is by using `psql` with the `-E` flag to connect to the machine in question and then use the `\password` command:
```
$ psql -E
psql (16.3)
Type "help" for help.

postgres=# \password myuser
Enter new password for user "myuser":
Enter it again:
********* QUERY **********
ALTER USER myuser PASSWORD 'SCRAM-SHA-256$4096:FP5Q/0n7Kr50KiZxt4zNnw==$UfhX6J1jBxsbWyxZPu1N85FfsGCjBY/MoJ4rmTwTBus=:zHuNlJhHcCwCs+7Jz+uxAcQqksObiUphAzPcbeTk7aA='
**************************
```

Reply:
Beautiful, so no need for custom scripts or local instance postgres instalation. Going to retest it and then I am sending an updated PR. Thanks @JelteF !

Reply:
Done. Thanks!